ambition_utils/rrule/models.py

```diff
@@ -313,9 +313,9 @@ def save(self, *args, **kwargs):
         # Call the parent save method
         super().save(*args, **kwargs)
 
-    def generate_dates(self, num_dates=20):
+    def generate_dates(self, num_dates=20, start_date=None):
```

Comment:
Your change to the docstring is helpful. Can you also change the method name to something like `get_dates`?

Reply:
I do think `get_dates` is better but is it worth a breaking change?

Reply:
ugh, I guess since this is in ambition-utils, it's not as simple as just updating consumers in ambition. Nah, I guess it's not worth it.

Reply:
if you want to rename it to `get_dates` you can move the code to a new method with the same args, then put a warning log in the `generate_dates` then call `get_dates`. the warning log can say it is deprecated and will be removed in version x (next major version).

Reply:
I went ahead with the rename. Let me know what we're not aligned on. c0c9638